src/Article.jsx

```diff
@@ -1,10 +1,13 @@
 import React, { useState, useEffect } from "react";
 import { useParams } from "react-router-dom";
-import { getArticleById } from './utils/api';
+import { getArticleById, getCommentsByArticleId } from './utils/api';
+import CommentCard from './CommentCard';
+import './App.css';
 
 function Article() {
   const { article_id } = useParams();
   const [article, setArticle] = useState(null);
+  const [comments, setComments] = useState([]);
```

Comment:
Could we also have some loading state in here for when we're waiting for the comments?